distribution/docs/src/main/resources/_contents/_resources/content-directory-monitor-contents.adoc

```diff
@@ -72,6 +72,11 @@ To set multi-valued attributes, use a separate override for each value. For exam
 `*topic.keyword=PPI*` +
 `*topic.keyword=radiology*`
 
+.Blacklist
```

Comment:
This needs to be updated.  I think it would be worthwhile mention that the CDM now uses the system properties.